trackingC_setup.Rmd

```diff
@@ -226,7 +226,7 @@ tibble(lognorm = ecs_vals_lognorm, transformed = runlist$ECS) %>%
 
 Run the model and store outputs and tracking data.
 
-```{r functions, cache=TRUE, message=FALSE}
+```{r functions, message=FALSE, cache=TRUE}
```

Comment:
Trivial, but is there a reason why you changed this?

Reply:
Didn't mean to; I clicked on the code chunk gear option, and it automatically rearranged them 🤷